addons/account/controllers/terms.py

```diff
@@ -5,9 +5,17 @@
 from odoo.http import request
 
 
+def sitemap_terms(env, rule, qs):
+    if qs and qs.lower() not in '/terms':
```

Comment:
This comparison seems very very odd here, does it really make sense for a qs of `/t` to match?

Also `qs` reads like it's an HTTP query string (which admittedly is a fault of the caller function as well).

Reply:
That shouldn't be in this PR, there was probably an error when he rebased the branch, it seems to have been including some other commits :grimacing: 

But yes, when the user search `/t` or `/te` etc when creating a link, this page should be returned.

![image](https://user-images.githubusercontent.com/30048408/168078551-cae6408b-dab2-46a0-9844-2bf7e34474ad.png)

Reply:
Indeed, there was a problem with a rebase, that commit does not belong to this PR 😬.